orchestration/templates/process-table.yaml

```diff
@@ -0,0 +1,46 @@
+{{- $schemaImage := printf "%s:%s" .Values.argoTemplates.diffBQTable.schemaImageName (default "latest" .Chart.AppVersion) }}
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: process-table
+spec:
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: table-name
+          - name: gcs-bucket
+          - name: gcs-prefix
+          - name: staging-bq-project
+          - name: staging-bq-dataset
+          - name: jade-bq-project
+          - name: jade-bq-dataset
+      dag:
+        tasks:
+          # 1) diff table
+          - name: diff-table
+            # WHEN the smoke test is not being run
+            when: '{{ not .Values.smokeTest.enable }}'
```

Comment:
I would rather push the `when` to the other DAG, on `process-tables`. That way as more steps are added here we don't have to copy-paste this condition everywhere.